llvm/lib/MCCAS/MCCASObjectV1.cpp

```diff
@@ -478,51 +478,87 @@ Expected<uint64_t> materializeAbbrevFromTagImpl(MCCASReader &Reader,
   return Size + *MaybePaddingSize;
 }
 
+static Error materializeDebugInfoUnopt(MCCASReader &Reader,
```

Comment:
we called the other method a `Maybe`, but here we didn't. Is there a reason for that?